scripts/bwctest_osd.sh

```diff
@@ -5,9 +5,9 @@
 
 set -e
 
-. scripts/bwc/utils.sh
-. scripts/bwc/opensearch_service.sh
-. scripts/bwc/opensearch_dashboards_service.sh
+. scripts/common/utils.sh
```

Comment:
could we create an issue into adding these to: https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/packages/osd-test